### PR TITLE
Improve chat feature (Issue 237)

### PR DESCRIPTION
**Pull request by jordanjclay123**
Clarify name and host fields and add window telling if chat connection is successful

### CODE REVIEW

**Comment by LoganRickert** [comment]
I would make sure that none of these changes conflict with Michael's request ( pull #254 ). His request moves a lot of the chat code in `MenuController.java` to its own class.

**Comment by jordanjclay123** [comment]
I did check and did not find any conflicts. Thanks for bringing it up, though.

File: src/edu/wright/cs/raiderplanner/controller/MenuController.java [hunk 1]
```diff
@@ -275,10 +285,30 @@ public void main() {
...
+		return chatConnection;
+	}
+
+	/**
+	 * Sets the boolean variable of whether or not there is a successful chat connection.
+	 *
+	 * @param newChatConnection
+	 *            - the value to which to set chatConnection
+	 */
+	public void setChatConnection(boolean newChatConnection) {
```
Comment (rsanchez-wsu) [inline src/edu/wright/cs/raiderplanner/controller/MenuController.java]: The purpose of this setter method is not clear to me. Why would an outside class need to set the status of the chat connection? Is it not the responsibility of the controller to manage that?

File: src/edu/wright/cs/raiderplanner/controller/MenuController.java [hunk 2]
```diff
@@ -508,8 +544,8 @@ protected void updateItem(final Milestone item, final boolean empty) {
 					if (this.showNotification.getTranslateY() == 0) {
 						TranslateTransition closeNot = new TranslateTransition(new Duration(173),
 								notifications);
-						closeNot.setToY(-(notifications.getHeight() + this.navShadowRadius + 56
-								+ 17));
+						closeNot.setToY(
```
Comment (rsanchez-wsu) [inline src/edu/wright/cs/raiderplanner/controller/MenuController.java]: Why was this reformatted?

File: src/edu/wright/cs/raiderplanner/controller/MenuController.java [hunk 3]
```diff
@@ -627,8 +663,8 @@ public void loadCalendar() {
 			}
 		});
 		// Populate Agenda:
-		ArrayList<Event> calendarEvents =
-				MainController.getSpc().getPlanner().getCurrentStudyProfile().getCalendar();
+		ArrayList<Event> calendarEvents = MainController.getSpc().getPlanner()
+				.getCurrentStudyProfile().getCalendar();
```
Comment (rsanchez-wsu) [inline src/edu/wright/cs/raiderplanner/controller/MenuController.java]: Why was this reformatted?

File: src/edu/wright/cs/raiderplanner/controller/MenuController.java [hunk 4]
```diff
@@ -747,8 +782,8 @@ protected void updateItem(final StudyProfile item, final boolean empty) {
 					if (this.showNotification.getTranslateY() == 0) {
 						TranslateTransition closeNot = new TranslateTransition(new Duration(173),
 								notifications);
-						closeNot.setToY(-(notifications.getHeight() + this.navShadowRadius + 56
-								+ 17));
+						closeNot.setToY(
```
Comment (rsanchez-wsu) [inline src/edu/wright/cs/raiderplanner/controller/MenuController.java]: Why was this reformatted?

File: src/edu/wright/cs/raiderplanner/controller/MenuController.java [hunk 5]
```diff
@@ -822,8 +857,8 @@ public void loadModules() {
...
```
Comment (rsanchez-wsu) [inline src/edu/wright/cs/raiderplanner/controller/MenuController.java]: Why was this reformatted?

**Comment by jordanjclay123** [comment]
@MichaelPantoja and I are planning to meet about our pull requests on Friday. Also, I now better understand what the documentation and the commit messages are supposed to be. I plan to remove the setter you mentioned. All of the reformatting was from trying to fix checkstyle issues, for which I initially used the wrong set of rules.

**Comment by rsanchez-wsu** [comment]
@jordanjclay123 @MichaelPantoja if you will not have the pull request put together by the end of the week and you need an extension in Pilot just let me know. My preference is to give you more time and have a nice clean set of changes rather than try to untangle a bunch of conflicts.

**Comment by rsanchez-wsu** [comment]
@jordanjclay123 @MichaelPantoja I just realized that your pull requests (this one and #254) were for project 1. I am not sure what you were planning for project 2, but I think from my last comment it wasn't clear that what I was requesting was that you collaborate on putting this work together into a single pull request and that would count as project 2. You will receive full credit for project 1 based on the work already completed. I will just need each of you to close your initial pull requests when you are ready.

**Pull request closed without merging**